<launch>
    <node pkg="agrobot" type="start_server.py" name="start_server" output="screen"/>
    <node pkg="agrobot" type="log.py" name="log" output="screen"/>
    <node pkg="agrobot" type="setup.py" name="setup" output="screen"/>
    <node pkg="agrobot" type="priority_decider.py" name="priority_decider" output="screen"/>
    <node pkg="agrobot" type="command_center.py" name="command_center" output="screen"/>
    <node pkg="agrobot" type="control_simulation.py" name="control_simulation" output="screen"/>
    <node pkg="agrobot" type="get_robot_commands.py" name="get_robot_commands" output="screen"/>
    <node pkg="agrobot" type="run_simulation.sh" name="run_simulation" output="screen"/>
</launch>
